app/cdap/components/EntityCard/index.js

```diff
@@ -137,6 +137,7 @@ export default class EntityCard extends Component {
               className={classnames({ 'with-version': this.props.entity.version })}
               title={this.props.entity.id}
               data-cy={`${this.props.entity.id}-header`}
+              data-testid={`${this.props.entity.id}-header`}
```

Comment:
@mrahanjam we're switching over to using data-testid? if so why do we still need data-cy?

Reply:
Cypress tests are still being used in cloud build, once we move away from using cypress completely, we can remove data-cy tags.